Doc/Compatability-table.md

```diff
@@ -0,0 +1,142 @@
+| Physics2D      | _UnityEngine_.**PhysicsJobOptions2D**                           | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| Physics2D      | _UnityEngine_.**PhysicsScene2D**                                | ❌[⁽²⁾](#note-2) | ❓[⁽³⁾](#note-3) | ❌[⁽²⁾](#note-2)  |
+| Physics2D      | _UnityEngine_.**RaycastHit2D**                                  | ❌[⁽²⁾](#note-2) | ❓[⁽³⁾](#note-3) | ❌[⁽²⁾](#note-2)  |
+| Random         | _UnityEngine_.**Random.State**                                  | ✔               | ✔               | ✔                |
+| Rendering      | _UnityEngine.Rendering_.**BatchCullingContext**                 | ❌[⁽⁴⁾](#note-4) | ❔[⁽⁵⁾](#note-5) | ❌[⁽⁴⁾](#note-4)  |
+| Rendering      | _UnityEngine.Rendering_.**BatchVisibility**                     | ❌[⁽⁶⁾](#note-6) | ❔[⁽⁵⁾](#note-5) | ❌[⁽⁶⁾](#note-6)  |
+| RenderPipeline | _UnityEngine.Rendering_.**FilteringSettings**                   | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| RenderPipeline | _UnityEngine.Rendering_.**RenderQueueRange**                    | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| RenderPipeline | _UnityEngine.Rendering_.**SortingLayerRange**                   | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| RenderPipeline | _UnityEngine.Rendering_.**VisibleLight**                        | ❌[⁽²⁾](#note-2) | ❓[⁽³⁾](#note-3) | ❌[⁽²⁾](#note-2)  |
+| Scripting      | _UnityEngine_.**LayerMask**                                     | ✔               | ✔               | ✔                |
+| Scripting      | _UnityEngine_.**RangeInt**                                      | ✔               | ✔               | ✔                |
+| SpriteShape    | _UnityEngine.U2D_.**AngleRangeInfo**                            | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| SpriteShape    | _UnityEngine.U2D_.**ShapeControlPoint**                         | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+| SpriteShape    | _UnityEngine.U2D_.**SpriteShapeMetaData**                       | ✔               | ✔               | ✖[⁽¹⁾](#note-1)  |
+
+1. ✖<a name="note-1"></a> Does not need a custom converter. Works out of the
+    box as-is.
+
+2. ❌<a name="note-2"></a> Contains ID or reference to _UnityEngine_.**Object** and
+    will therefore never be deserializable.
+
+3. ❓<a name="note-3"></a> Serializing this type is discouraged. Serializing
+    this type will highly likely result in errors such as self referencing loops
+    or infinite recursions.
+
+4. ❌<a name="note-4"></a> Type directly or indirectly contains reference to
+    the NativeArray or NativeSlice types. Deserializing these types will cause
+    imminent memory leaks and so deserializing (writing JSON) using these types
+    are therefore highly discouraged.
+
+5. ❔<a name="note-5"></a> Serializing this type has not been tested nor
+    proven to work. It is fully possible they work alright, but be wary.
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461644)